### PR TITLE
[Console] allow answer to be trimmed by adding a flag

### PR DESCRIPTION
**Pull request by Simperfit**
| Q             | A
| ------------- | ---
| Branch?       | master 
| Bug fix?      | no
| New feature?  | yes 
| BC breaks?    | no     
| Deprecations? | no 
| Tests pass?   | yes    
| Fixed tickets | none   
| License       | MIT
| Doc PR        | https://github.com/symfony/symfony-docs/pull/11603 

According to https://github.com/symfony/symfony/issues/23210#issuecomment-495998087 we add a new flag in the `Question` class to be able to not trim the answer.

### CODE REVIEW

**Comment by Simperfit** [comment]
fabbot failure is a false positive

File: src/Symfony/Component/Console/Tests/Helper/QuestionHelperTest.php [hunk 1]
```diff
@@ -198,6 +212,40 @@ public function testAskWithAutocomplete()
...
+        $this->assertEquals('AsseticBundleTest', $dialog->ask($this->createStreamableInputInterfaceMock($inputStream), $this->createOutputInterface(), $question));
+        $this->assertEquals('FrameworkBundle', $dialog->ask($this->createStreamableInputInterfaceMock($inputStream), $this->createOutputInterface(), $question));
+        $this->assertEquals('SecurityBundle', $dialog->ask($this->createStreamableInputInterfaceMock($inputStream), $this->createOutputInterface(), $question));
+        $this->assertEquals('FooBundleTest', $dialog->ask($this->createStreamableInputInterfaceMock($inputStream), $this->createOutputInterface(), $question));
+        $this->assertEquals('AcmeDemoBundle ', $dialog->ask($this->createStreamableInputInterfaceMock($inputStream), $this->createOutputInterface(), $question));
+        $this->assertEquals('AsseticBundle', $dialog->ask($this->createStreamableInputInterfaceMock($inputStream), $this->createOutputInterface(), $question));
+        $this->assertEquals('FooBundle', $dialog->ask($this->createStreamableInputInterfaceMock($inputStream), $this->createOutputInterface(), $question));
```
Comment (chalasr) [inline src/Symfony/Component/Console/Tests/Helper/QuestionHelperTest.php]: Some answers should contain trailing/leading whitespaces for the test to be relevant

**Comment by Simperfit** [comment]
@chalasr update to 4.4 and review fixed

File: src/Symfony/Component/Console/CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+4.4.0
+-----
+
+* added `Question::setTrimmable` default to true to allow the anwser to be trimmed or not
```
Comment (apfelbox) [inline src/Symfony/Component/Console/CHANGELOG.md]:
```suggestion
* added `Question::setTrimmable` default to true to allow the answer to be trimmed or not
```

**Comment by Simperfit** [comment]
AppVeyor failure is unrelated, PR rebased

File: src/Symfony/Component/Console/CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+4.4.0
+-----
+
+* added `Question::setTrimmable` default to true to allow the answer to be trimmed or not
```
Comment (fabpot) [inline src/Symfony/Component/Console/CHANGELOG.md]: Let's remove the `or not`

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @Simperfit.

**Pull request merged**